src/app/collections/collection.html

```diff
@@ -125,6 +124,11 @@
           <a href ng-click="vm.triggerClearFilters()" ng-if="vm.isCategoryChanged()">Clear Filters</a>
         </div>
       </div> <!-- #filters -->
+      <div class="form group" ng-if="vm.sedDownloadAllButton()">
```

Comment:
This whole section, including the stuff in the controller, should be in the `collections/sed/`component somewhere. The stuff that goes in the collections directive is supposed to be stuff that could be used by multiple collections, but a button to download SED details isn't like that